Automattic/wp-calypso
# Add Google Adwords card to Marketing Tools

#### Changes proposed in this Pull Request

* Adds Google Adwords card to marketing tools page

#### Testing instructions

* Have site with plan lower than premium
* Goto to Marketing Tools page
* Observe Button on Adwords card says "Upgrade To Premium"
* Have another site with Premium or higher
* Observe button says "Generate Code" and has fine print
* Click generate code observe ToS dialog opens
* Click cancel and observe it closes
* Open again and accept
* Observe that you now code with copy button
* Copy it to ensure that works
* You should also see fine print with a link
* Ensure link works
* Click "Setup Google Ads" button to ensure that works
![Screen Shot 2019-04-30 at 3 30 25 PM](https://user-images.githubusercontent.com/6817400/56989924-a96cf280-6b61-11e9-9334-8ed1434c1789.png)
![Screen Shot 2019-04-30 at 3 30 39 PM](https://user-images.githubusercontent.com/6817400/56989926-a96cf280-6b61-11e9-930d-a0e022154976.png)
![Screen Shot 2019-04-30 at 3 30 49 PM](https://user-images.githubusercontent.com/6817400/56989927-a96cf280-6b61-11e9-88cb-984d3e3cf4f6.png)
![Screen Shot 2019-04-30 at 4 04 18 PM](https://user-images.githubusercontent.com/6817400/56989928-a96cf280-6b61-11e9-9e9c-5ff99d915908.png)
